geode/advanced-examples/cascading-functions/src/main/java/example/springdata/geode/functions/cascading/Customer.java

```diff
@@ -0,0 +1,80 @@
+/*
+ *
+ *  * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ *  * agreements. See the NOTICE file distributed with this work for additional information regarding
+ *  * copyright ownership. The ASF licenses this file to you under the Apache License, Version 2.0 (the
+ *  * "License"); you may not use this file except in compliance with the License. You may obtain a
+ *  * copy of the License at
+ *  *
+ *  * http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  * Unless required by applicable law or agreed to in writing, software distributed under the License
+ *  * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ *  * or implied. See the License for the specific language governing permissions and limitations under
+ *  * the License.
+ *
+ */
+
+package example.springdata.geode.functions.cascading;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.gemfire.mapping.annotation.Region;
+
+import java.io.Serializable;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * A customer used for Lucene examples.
+ *
+ * @author Udo Kohlmeyer
+ * @author Patrick Johnson
+ */
+
+@Region(name = "Customers")
+public class Customer implements Serializable {
```

Comment:
We're making use of Project Lombok in our examples to not pollute the code with getters/setters/equals/hashCode methods. Using Lombok for all of the domain types would help in reducing the boilerplate code to the necessary bits.

Reply:
Data objects now use Lombok.